cmd/admin/unpauseAccount.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+import (
+	"bufio"
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+	"strconv"
+
+	sapb "github.com/letsencrypt/boulder/sa/proto"
+)
+
+// subcommandUnpauseAccount encapsulates the "admin unpause-account" command.
+type subcommandUnpauseAccount struct {
+	batchFile string
+	regID     int64
+}
+
+var _ subcommand = (*subcommandUnpauseAccount)(nil)
+
+func (u *subcommandUnpauseAccount) Desc() string {
+	return "Administratively unpause an account to allow certificate issuance attempts"
+}
+
+func (u *subcommandUnpauseAccount) Flags(flag *flag.FlagSet) {
+	flag.StringVar(&u.batchFile, "batch-file", "", "Path to a file containing multiple account IDs where each is separated by a newline")
+	flag.Int64Var(&u.regID, "account", 0, "A single account ID to unpause")
+}
+
+func (u *subcommandUnpauseAccount) Run(ctx context.Context, a *admin) error {
+	if u.batchFile == "" && u.regID == 0 {
+		return errors.New("either the -batch-file or -account flag is required")
+	}
+	if u.batchFile != "" && u.regID != 0 {
+		return errors.New("only one of -batch-file or -account flag should be used")
+	}
+
+	var regIDs []int64
+	var err error
+	if u.batchFile != "" {
+		regIDs, err = a.readUnpauseAccountFile(u.batchFile)
+		if err != nil {
+			return err
+		}
+	} else {
+		regIDs = []int64{u.regID}
+	}
+
+	err = a.unpauseAccounts(regIDs)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// unpauseAccount allows administratively unpausing all identifiers for an
+// account.
+func (a *admin) unpauseAccounts(regIDs []int64) error {
```

Comment:
This function should take a context as its argument, so it doesn't have to create one from scratch.